File: app/src/main/java/fr/atech/robotcom/ui/configuration/ConfigurationViewModel.java
package fr.atech.robotcom.ui.configuration;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ConfigurationViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public ConfigurationViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is dashboard fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/fr/atech/robotcom/ui/configuration/ConfigurationViewModel.kt
package fr.atech.robotcom.ui.configuration

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ConfigurationViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is configuration Fragment"
    }
    val text: LiveData<String> = _text
}